compiler/rustc_transmute/src/maybe_transmutable/tests.rs

```diff
@@ -107,7 +107,7 @@ mod bool {
             #[cfg(feature = "rustc")]
             let mut set = crate::Set::default();
             #[cfg(not(feature = "rustc"))]
-            let mut set = std::collections::HashSet::new();
+            let mut set = rustc_data_structures::fx::FxHashSet::default();
```

Comment:
Also how is this change related?

Reply:
rustc runs clippy that lints against using bare `HashSet`, upstream doesnt catch this because they dont build without the `rustc` feature.

This change will be upstreamed afterwards.